docs/UserGuide.md

```diff
@@ -115,7 +131,6 @@ The features are categorised into the different categories as follows:
 --------------------------------------------------------------------------------------------------------------------
 
 ### Overview of the Command Format
-This section explains the command format used for TAB's commands.<br/>
```

Comment:
I feel like we can leave this here? General overview of the section is clearer isn't it？